test/ci-operator-integration/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+WORKDIR="$( mktemp -d )"
+trap 'rm -rf "${WORKDIR}"' EXIT
+
+readonly TEST_ROOT="$( dirname "${BASH_SOURCE[0]}")"
+readonly EXPECTED="${TEST_ROOT}/expected_files/expected.json"
+readonly EXPECTED_WITH_TEMPLATE="${TEST_ROOT}/expected_files/expected_with_template.json"
+readonly DRY_RUN_JSON="${WORKDIR}/ci-op-dry.json"
+readonly DRY_RUN_WITH_TEMPLATE_JSON="${WORKDIR}/ci-op-template-dry.json"
+
+echo "[INFO] Running ci-operator in dry-mode..."
+if ! ci-operator --dry-run --namespace testns --config ${TEST_ROOT}/config/test-config.yaml  --git-ref=openshift/origin@master 2>&1 |  grep -v "^[0-9]" > ${DRY_RUN_JSON}; then
```

Comment:
I think I'd prefer us to use a `$JOB_SPEC` -- I don't really care how well we support a `--git-ref`

Reply:
We should not `grep -v` here but instead use `diff -I`

Reply:
addressed